Kconfig

```diff
@@ -4,11 +4,11 @@ mainmenu "SOF $(PROJECTVERSION) Configuration"
 
 comment "Compiler: $(CC_VERSION_TEXT)"
 
+if CAVS
```

Comment:
"if" for visible purposes should be used directly on "config X" options. If you want to hide a whole menu then it should be done by "visible if CAVS" menu parameter

Reply:
`visible if` will make it work like configs would have optional prompts, while `if` will add `depends on` for all of them. But you motivated me to do something better, I can just add `depends on CAVS` for `menu` and it will have the same effect without wrapping with ifs.